src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -373,15 +373,27 @@ void Renderer::Impl::render(const UpdateParameters& updateParameters) {
         }
     }
 
-    auto newPlacement = std::make_unique<Placement>(parameters.state, parameters.mapMode);
-    for (auto it = order.rbegin(); it != order.rend(); ++it) {
-        if (it->layer.is<RenderSymbolLayer>()) {
-            newPlacement->placeLayer(*it->layer.as<RenderSymbolLayer>(), parameters.projMatrix, parameters.debugOptions & MapDebugOptions::Collision);
+    bool placementChanged = false;
+    if (placement->recentUntil <= parameters.timePoint) {
```

Comment:
In still mode, we should always do placement.

Reply:
fixed